source/MRMesh/MRMeshLoadStep.cpp

```diff
@@ -32,6 +31,23 @@
 #include <opencascade/TopoDS.hxx>
 #pragma warning( pop )
 
+#if ( OCC_VERSION_MAJOR < 7 ) || ( OCC_VERSION_MAJOR == 7 && OCC_VERSION_MINOR < 5 )
+namespace MR::MeshLoad
+{
+
+Expected<std::shared_ptr<Object>, std::string> fromSceneStepFile( const std::filesystem::path& path, const MeshLoadSettings& settings /*= {}*/ )
+{
+    // TODO: support OpenCASCADE 7.3
+    return unexpected( "Unsupported: outdated OpenCASCADE version" );
```

Comment:
May be build without Opencascade on that system?
And fail the compilation if attempt to enable Opencascade.